### ISSUE TITLE
IPv6 Cluster blanks out UI / throws NPEs into logs

### ISSUE THREAD

**Issue by Rooks103** [thelastpickle/cassandra-reaper#1025]
Added a cluster that uses IPv6 into one of our Reaper instances and shortly after, the UI went blank and saw a bunch of failed connection messages in the logs. This happened with both Reaper 2.2.1 and Reaper 2.1.3 (although 2.1.3 was also with Java 11 which has since been fixed and moved back to Java 8).

Reaper is setup with a Cassandra backend, however, the IPv6 cluster is not the backend cluster. Clusters are using SSL on the DB, but just using authentication on JMX. 

```
WARN   [ClusterResource] c.d.d.c.p.EC2MultiRegionAddressTranslator - Error resolving null, returning it as-is
java.lang.NullPointerException: null
        at com.datastax.driver.core.policies.EC2MultiRegionAddressTranslator.reverse(EC2MultiRegionAddressTranslator.java:128)
        at com.datastax.driver.core.policies.EC2MultiRegionAddressTranslator.translate(EC2MultiRegionAddressTranslator.java:91)
        at io.cassandrareaper.jmx.JmxProxyImpl.connect(JmxProxyImpl.java:213)
        at io.cassandrareaper.jmx.JmxProxyImpl.connect(JmxProxyImpl.java:179)
        at io.cassandrareaper.jmx.JmxConnectionFactory$JmxConnectionProvider.apply(JmxConnectionFactory.java:231)
        at java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1660)
        at io.cassandrareaper.jmx.JmxConnectionFactory.connectImpl(JmxConnectionFactory.java:103)
        at io.cassandrareaper.jmx.JmxConnectionFactory.connectAny(JmxConnectionFactory.java:138)
        at io.cassandrareaper.jmx.ClusterFacade.connectImpl(ClusterFacade.java:847)
        at io.cassandrareaper.jmx.ClusterFacade.connect(ClusterFacade.java:831)
        at io.cassandrareaper.jmx.ClusterFacade.getNodesStatus(ClusterFacade.java:208)
        at io.cassandrareaper.resources.ClusterResource.lambda$getEndpointState$4(ClusterResource.java:429)
        at com.codahale.metrics.InstrumentedExecutorService$InstrumentedCallable.call(InstrumentedExecutorService.java:202)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
ERROR  [ClusterResource] i.c.j.JmxConnectionFactory - Failed creating a new JMX connection to 2600:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx:7199
java.lang.NullPointerException: null
        at io.cassandrareaper.jmx.JmxProxyImpl.connect(JmxProxyImpl.java:213)
        at io.cassandrareaper.jmx.JmxProxyImpl.connect(JmxProxyImpl.java:179)
        at io.cassandrareaper.jmx.JmxConnectionFactory$JmxConnectionProvider.apply(JmxConnectionFactory.java:231)
        at java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1660)
        at io.cassandrareaper.jmx.JmxConnectionFactory.connectImpl(JmxConnectionFactory.java:103)
        at io.cassandrareaper.jmx.JmxConnectionFactory.connectAny(JmxConnectionFactory.java:138)
        at io.cassandrareaper.jmx.ClusterFacade.connectImpl(ClusterFacade.java:847)
        at io.cassandrareaper.jmx.ClusterFacade.connect(ClusterFacade.java:831)
        at io.cassandrareaper.jmx.ClusterFacade.getNodesStatus(ClusterFacade.java:208)
        at io.cassandrareaper.resources.ClusterResource.lambda$getEndpointState$4(ClusterResource.java:429)
        at com.codahale.metrics.InstrumentedExecutorService$InstrumentedCallable.call(InstrumentedExecutorService.java:202)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)

**Comment by cmvelo** [comment]
👋 — just checking to see when this issue might be looked at?   We have a (fairly large) IPv6 ring badly in need of repair and it would be highly preferable to use reaper instead attempting to do it manually.

**Comment by Rooks103** [comment]
This should be fixed with MR #1056. Closing issue.
